### DISCUSSION TITLE
ClipboardService change '\t' to ' ' in copy content after publish into IIS

### DISCUSSION THREAD

**Discussion by brucetju** [dotnetcore/BootstrapBlazor#2998] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

I use clipboardService for users to copy infos with '\t' into excel directly. It can work well during debugging, but failed after published to IIS. It seems something auto changed '\t' to ' '. I am not sure the reason, could you help on this?

### Expected Behavior

copy same thing which contains ‘\t’ to clipboard

### .NET Version

6.0

**Comment by bb-auto** [comment]
@brucetju Thank you for reporting. We will give triage later.

**Comment by ArgoZhang** [comment]
@brucetju needs more information. could you provider a mini-repro project?

**Comment by brucetju** [comment]
[ChipboardTest.zip](https://github.com/dotnetcore/BootstrapBlazor/files/14403205/ChipboardTest.zip)

I created a brand new project with Blazor app template based on net6.0 as attached. I added a button to copy current datetime to clipboard with '\t'.

Step1: During debugging, copy by clicking button, then paste into excel and result shows like this:(correct, as I expected)
![图片](https://github.com/dotnetcore/BootstrapBlazor/assets/61259114/0639dce4-e59a-4a8b-ae4f-3436aa4bcb5c)

Step2: Publish to folder and config in IIS
![图片](https://github.com/dotnetcore/BootstrapBlazor/assets/61259114/3edf5742-a801-49f6-a4be-a798f5bff321)

Step3: open IIS website and do same thing as Step1, but different result is showed like following. It seems '\t' is replaced by ' '.
![图片](https://github.com/dotnetcore/BootstrapBlazor/assets/61259114/82a94197-45d2-44b2-a6cb-1920e642d411)

Any further details, please contact to me directly.

**Comment by ArgoZhang** [comment, marked as answer]
@brucetju 

我测试了一下，我的环境里没问题的。你按照下面的办法试一下

1. 正常发布到 publish 文件夹
![image](https://github.com/dotnetcore/BootstrapBlazor/assets/22001478/4ed80776-69aa-414d-85c4-90689d77a595)

2. 双击 `ChipboardTest.exe`
3. 根据你 Issue 中的操作

我这里时没问题的。稍后我找一个 IIS 发布一下试一下，再回复

### IIS
都正常，没问题
